python/ray/serve/api.py

```diff
@@ -53,8 +53,15 @@ def shutdown_serve_client():
 
     def __del__(self):
         if not self._detached:
+            logger.info("Shutting down Ray Serve because client went out of "
+                        "scope.")
```

Comment:
```suggestion
            logger.info("Shutting down Serve instance because client went out of "
                        "scope. To prevent this, either keep a reference to the client object or use serve.start(detached=True).")
```